### PR TITLE
Setup codebase for frontend

### PR DESCRIPTION
**Pull request by okolipraiz**
This is the code base setup, we are going with bootstrap and next js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+pnpm dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+
+[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+
+The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
```
Comment (CeoFred) [inline README.md:21]: Let's not have this whole auto-generated nextjs content here

File: pages/_app.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                      content="width=device-width, initial-scale=1"
+                  />
+                  <link rel="icon" href="/favicon.ico" />
+              </Head>
+              <Component {...pageProps} />
+              <Script
+                  src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
+                  integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4"
+                  crossorigin="anonymous"
+              />
```
Comment (CeoFred) [inline pages/_app.js:35]: Why importing via cdn? does this make things any faster or what's the reason?

File: pages/_document.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+export default function Document() {
+    return (
+        <Html lang="en">
+            <Head />
+            <link
+                href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+                rel="stylesheet"
+                integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+                crossorigin="anonymous"
+            />
```
Comment (CeoFred) [inline pages/_document.js:12]: This could have been on the same file where you insert scripts assuming using CDN is the best way to import boostrap.

**Review by CeoFred — CHANGES_REQUESTED** [review]

File: pages/_document.js (hunk 3 above)
Comment (okolipraiz) [reply pages/_document.js:12]: It doesn’t work that way when using bootstrap CDN

File: pages/_app.js (hunk 2 above)
Comment (okolipraiz) [reply pages/_app.js:35]: Firstly if we use the bootstrap downloaded bundle it caches a whole lot of css files that are not necessary, and it’s faster too sir.

File: README.md (hunk 1 above)
Comment (okolipraiz) [reply README.md:21]: I felt it was necessary for orders to use and setup their projects too

Comment (okolipraiz) [reply README.md:21]: I will do a quick change on it to something more standard

**Pull request closed without merging**